### PR TITLE
Support training time series models with multiple output time points on PAI

### PR DESCRIPTION
**Pull request by typhoonzero**
Support training time series model with multiple label value like: `0.32,0.14,0.23`.

TODO: add a test case

### CODE REVIEW

File: pkg/sql/codegen/tensorflow/codegen.go [hunk 1]
```diff
@@ -326,7 +326,8 @@ func InitializeAttributes(trainStmt *ir.TrainStmt) error {
 		// Special attributes defined as global variables in `sqlflow_models`
 		modelAttr.Update(attribute.Dictionary{
 			"model.optimizer": {attribute.Unknown, nil, "Specify optimizer", nil},
-			"model.loss":      {attribute.Unknown, nil, "Specify loss", nil}})
+			"model.loss":      {attribute.Unknown, nil, "Specify loss", nil},
+			"model.*":         {attribute.Unknown, nil, "Any model parameters defined in custom models", nil}})
```
Comment (typhoonzero) [inline pkg/sql/codegen/tensorflow/codegen.go:330]: I added this line so that the attribute checker will recognize newly added parameters in the custom Keras model

Comment (shendiaomo) [reply pkg/sql/codegen/tensorflow/codegen.go:330]: Do we have a better solution? `model.*` actually disables the attribute checker and defer the diagnostic from compile-time to run-time, which makes it harder for a user to debug her SQL program.

Comment (typhoonzero) [reply pkg/sql/codegen/tensorflow/codegen.go:330]: I'm afraid there may be no better solutions, people still need to develop a new model and test it with SQLFlow before SQLFlow can generate those attribute checkers.

File: python/sqlflow_submitter/tensorflow/input_fn.py [hunk 2]
```diff
@@ -147,7 +159,12 @@ def pai_dataset(table,
     ]
     if label_spec and label_spec["feature_name"]:
         selected_cols.append(label_spec["feature_name"])
-        dtypes.append(label_spec["dtype"])
+        # print("reading from paiio label name %s label dtype: %s" %
```
Comment (Yancey0623) [inline python/sqlflow_submitter/tensorflow/input_fn.py]: Can remove the commented code.

**Review by Yancey0623 — DISMISSED** [review]

**Review by Yancey0623 — APPROVED** [review]
LGTM
